src/components/AdminSettings/AdminSettingsMaster.jsx

```diff
@@ -1,13 +1,38 @@
+import { useState, useEffect } from 'react';
 import { Tabs, TabList, TabPanels, Tab, TabPanel, Text } from '@chakra-ui/react';
 import AdminsTable from './AdminsTable.jsx';
 import AdminAccount from './AdminAccount.jsx';
+import { useAuth } from '../../contexts/AuthContext.jsx';
+import { useBackend } from '../../contexts/BackendContext.jsx';
+import { useNavigate } from 'react-router-dom';
+import NotificationsDrawer from '../NotificationsDrawer/NotificationsDrawer.jsx';
 import classes from './AdminSettings.module.css';
 
 const AdminSettingsMaster = () => {
+  const { isAdmin } = useAuth();
+  const navigate = useNavigate();
+  const [notification, setNotification] = useState([]);
+  const { backend } = useBackend();
+
+  useEffect(() => {
+    const checkIsAdmin = async () => {
+      if (!(await isAdmin())) {
+        navigate('/BusinessDashboard');
+      }
+    };
+    checkIsAdmin();
+    const fetchNotifications = async () => {
+      const response = await backend.get('/notification/0');
+      setNotification(response.data);
+    }
+    fetchNotifications();
+  }, [isAdmin, navigate]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'backend'. Either include it or remove the dependency array.